cypress/e2e/undo-redo/block-configs-should-update-flow-state/mcq-block-configs.cy.js

```diff
@@ -0,0 +1,176 @@
+describe('Mutate flow state for the case (branching) block', () => {
+  const CHOICE_NAME = 'Foo'
+  const SECOND_CHOICE_NAME = 'Bar'
+
+  describe('common mcq block configs', () => {
+    beforeEach(() => {
+      cy.createFlow({
+        label: 'mcq block test',
+      })
+
+      cy.addBlock(['Select One Response'])
+
+      cy.get('[data-cy="choices-builder"]')
+        .find('[data-cy="resource--text"]')
+        .last()
```

Comment:
Can we avoid `last()` ? this is a bit fragile, especially if we reorganize the UI. I'd suggest we add a dynamic `data-cy` on choices like ```data-cy="`choices-${uuid}`"``` or something. Your thoughts

Reply:
It's literally the last element, though I added `data-cy="choices-builder--editor-new"` to make it as explicit as possible.

Reply:
I see what you mean, sorry about the confusion. Thanks for making it more explicit